src/app/projects.js

```diff
@@ -0,0 +1,36 @@
+import { createOptions } from './todoDom';
+
+class Project {
+  constructor(name) {
+    this.name = name;
+  }
+}
+
+let projects = JSON.parse(window.localStorage.getItem('projects'));
+
+const initialSetup = () => {
+  if (projects === null) {
+    projects = [];
+    const project = new Project('All todos');
+    projects.push(project);
+    localStorage.setItem('projects', JSON.stringify(projects));
+    createOptions();
+  }
+};
+
+initialSetup();
+
+const projectForm = () => {
+  const submit = document.getElementById('id-project-form');
+  submit.onsubmit = () => {
+    const inputValue = document.getElementById('project-name');
+    const project = new Project(inputValue.value);
+    projects.push(project);
+    document.getElementById('id-project-form').reset();
+    localStorage.setItem('projects', JSON.stringify(projects));
+    createOptions();
+    window.location.reload();
+    return false;
+  };
+};
+export default projectForm;
```

Comment:
Separate DOM manipulation from Logic. 
- Calling `projects.push` is Logic
- `localStorage.setItem` is Logic
The code between the two are DOM manipulation.  SEPARTE IT.

Kindly, read the comments left about TODO.